src/routines/routines.ycp

```diff
@@ -912,12 +912,12 @@ define list<string> GetAllInterfaces()
 
 boolean SetLinkUp( string dev_name)
 {
-    return Run( sformat("ip link set %1 up", dev_name) );
+    return Run( sformat("ifup %1", dev_name) );
 }
 
 boolean SetLinkDown( string dev_name)
 {
-    return Run( sformat("ip link set %1 down", dev_name) );
+    return Run( sformat("ifdown %1", dev_name) );
```

Comment:
"Use complete interface up / down instead of link up / down " describes the WHAT of the change but not the WHY.

Besides, the functions are explicitly named Set**Link**Up/Down so it is wrong to make them do something else.